src/main/java/uk/gov/hmcts/reform/prl/enums/bundle/BundlingDocGroupEnum.java

```diff
@@ -32,6 +32,9 @@ public enum BundlingDocGroupEnum {
     @JsonProperty("respondentApplication")
     respondentApplication("respondentApplication", "respondentApplication"),
 
+    @JsonProperty("ordersFromOtherProceedings")
```

Comment:
which usecase covers this change?

Reply:
This change is not required as part of this ticket, so removed it.